mongo_connector/doc_managers/neo4j_doc_manager.py

```diff
@@ -33,13 +31,20 @@ class DocManager(DocManagerBase):
   def __init__(self, url, auto_commit_interval=DEFAULT_COMMIT_INTERVAL,
                  unique_key='_id', chunk_size=DEFAULT_MAX_BULK, **kwargs):
     self.remote_graph = Graph(url, **kwargs.get('clientOptions', {}))
-    graph = Graph()
+    self.graph = Graph()
     self.auto_commit_interval = auto_commit_interval
     self.unique_key = unique_key
     self.chunk_size = chunk_size
     if self.auto_commit_interval not in [None, 0]:
       self.run_auto_commit()
-  
+    self._formatter = DefaultDocumentFormatter()
+    self.query_nodes = {}
```

Comment:
Is there a new instance of doc-manager created for each update call? And what about concurrency on these calls?
Otherwise the variables should probably be method local and passed on?